install/src/centreon-apache-https.conf

```
@@ -0,0 +1,75 @@
+#
+# Section added by Centreon Install Setup
+#
+Header set X-Frame-Options: "sameorigin"
+Header always edit Set-Cookie ^(.*)$ $1;HttpOnly;Secure
+ServerSignature Off
+ServerTokens Prod
+TraceEnable Off
+
+Alias /centreon/api @CENTREON_INSTALL_DIR@
+Alias /centreon @CENTREON_INSTALL_DIR@/www/
+
+<LocationMatch ^/centreon/(?!api/latest/|api/beta/|api/v[0-9]+/|api/v[0-9]+\.[0-9]+/)(.*\.php(/.*)?)$>
```

```suggestion
<LocationMatch ^/centreon/(authentication|api/(latest|beta|v[0-9]+|v[0-9]+\.[0-9]+))/.*$>
```
Shouldnt we add the authentication path ?